### DISCUSSION TITLE
Compile with -flto

### DISCUSSION THREAD

**Discussion by dario-loi** [cabinpkg/cabin#829] [Ideas]
Currently the release flags optimize aggressively through -O3, but they omit link time optimization!

I compiled from source with `-flto` enabled and the executable size of poac went down from 876 KB to 748 KB, it is reasonable to assume that this also leads to a speedup in the tool itself, however, I did not run benchmarks.

This could be an easy performance win for release builds of `poac`.

**Comment by ken-matsui** [comment]
I will work on this. I'll let you know once this is implemented.

**Comment by ken-matsui** [comment]
This is implemented in https://github.com/poac-dev/poac/pull/832.
